package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "multi-select-react",
-  "version": "0.1.8",
+  "version": "0.2.1",
```

Comment:
Try not to change version in package.json. It help to keep track of versions. Version in package.json should be same as the branch name.

Reply:
OK,I will revert it back.

Reply:
Fixed and Pushed a new commit